src/reference/00-Getting-Started/09-Multi-Project.md

```diff
@@ -10,65 +10,68 @@ out: Multi-Project.html
 Multi-project builds
 --------------------
 
-This page introduces multiple projects in a single build.
+This page introduces multiple subprojects in a single build.
 
 Please read the earlier pages in the Getting Started Guide first, in
 particular you need to understand [build.sbt][Basic-Def] before reading
 this page.
 
-### Multiple projects
+### Multiple subprojects
 
-It can be useful to keep multiple related projects in a single build,
+It can be useful to keep multiple related subprojects in a single build,
 especially if they depend on one another and you tend to modify them
 together.
 
-Each sub-project in a build has its own source directories, generates
+Each subproject in a build has its own source directories, generates
 its own jar file when you run package, and in general works like any
 other project.
 
 A project is defined by declaring a lazy val of type
 [Project](../api/sbt/Project.html). For example, :
 
 ```scala
-lazy val util = project
+lazy val util = (project in file("util"))
 
-lazy val core = project
+lazy val core = (project in file("core"))
 ```
 
-The name of the val is used as the project's ID and base directory name.
-The ID is used to refer to the project at the command line. The base
-directory may be changed from the default using the in method. For
-example, the following is a more explicit way to write the previous
-example:
+The name of the val is used as the subproject's ID, which
+is used to refer to the subproject at the sbt shell.
+
+Optionally the base directory may be omitted if it's same as the name of the val.
 
 ```scala
-lazy val util = project.in(file("util"))
+lazy val util = project
 
-lazy val core = project in file("core")
+lazy val core = project
 ```
 
-#### Common settings
+#### Build settings
 
 To factor out common settings across multiple projects,
 create a sequence named `commonSettings` and call `settings` method
 on each project. Note `_*` is required to pass sequence into a vararg
 method.
 
 ```scala
-lazy val commonSettings = Seq(
-  organization := "com.example",
-  version := "0.1.0",
-  scalaVersion := "$example_scala_version$"
-)
+lazy val root = (project in file("."))
```

Comment:
I suggest that the recommended default style for multi-project builds looks like this

```scala
organization in ThisBuild := "com.foo"
scalaVersion in ThisBuild := "2.12.1"
version      in ThisBuild := "0.1.0"
// Optionally, use inThisBuild(List(...)) instead of repeating `in ThisBuild`.
// I personally think `in ThisBuild` reads cleaner but I'm happy to use
// inThisBuild for the sake of following convention.

lazy val core = project
  .settings(
    // ...
  )
  .enablePlugins(
    // ..
  )

lazy val extension = project
  .settings(
    // ...
  )
  .dependsOn(
    core
  )
```
Most notably, there is no `root` project. Why?

- Projects are automatically aggregated, which keeps things DRY. It's annoying to create a new project and start hacking with `~compile/test` only to realize you forgot to aggregate your new project. I imagine this can be especially confusing for beginners. It's rare that I don't want to exclude a project from aggregation, and I would consider it to be an advanced use-case.
- The root project name automatically becomes the root directory name. The root project name is used in IDEs like IntelliJ. My `File > Open recent` list is currently clogged projects named `root`.